src/agent/cmd/main.go

```diff
@@ -61,30 +61,36 @@ func main() {
 	}
 
 	deployer := deployer.New(*apiServerAddr, cfg.nodeName, cfg.podID)
-	err = retry.ExpBackOffWithLimit(func() error {
+	for {
+		communicateWithNode(cfg.nodeName, deployer)
+	}
+
+	deployer.Stop()
+	log.Infof("Hello deployer\n")
```

Comment:
This log msg seems, Can we change this to another clear message?

Reply:
done